src/Explore/Simple.js

```diff
@@ -17,7 +19,16 @@ function Simple({ summary, releaseDate }) {
       </Box>
       <br />
       <Text sx={{ fontStyle: 'italic' }}>
-        Released: {parseDate(releaseDate)}
+        Released on{' '}
+        {provider
+          ? `${formatDateVerbose(releaseDate)} by the
```

Comment:
I would maybe remove `the`, since "by the MAXIV doesn't sound great, while "by <provider>" is always okay. Maybe also replace the line break with a space unless there's a reason I'm not getting?